omnigibson/prims/geom_prim.py

```diff
@@ -50,8 +50,9 @@ def _post_load(self):
         # run super first
         super()._post_load()
 
-        # By default, GeomPrim shows up in the rendering.
-        self.purpose = "default"
+        # Temporarily set the opacity threshold to be 0.1. This will be automated in later exports.
+        if self.material:
+            self.material.opacity_threshold = 0.1
```

Comment:
Can you add some comments? e.g. ray traicing can only handle binary opacity using `opacity_threshold`